sites/topomath.asu.edu/modules/system_descriptions/static/drupal_root

```diff
@@ -0,0 +1 @@
+/Applications/XAMPP/htdocs/LMS
```

Comment:
We have a similar file on server as well since dragoon LMS. On server we have a different path for drupal root. The connected issue is we have a php file to which we send ajax requests (nonClassUpdates.php ) which updates the LMS db according to different requests we make on non class models. This was the solution I came up initially when we made dragoon LMS in initial stages of learning drupal. I am sure there would be a better way to handle requests using drupal API and making a separate module for handling such requests. I will ask jon to make a separate task for this. This answers the below question as in why we are not using .module file